problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -58,6 +104,16 @@
 			\item The plane $\mathcal Q\subseteq\R^3$ passing through the points
 			$D=(1,1,1)$, $E=(1,-2,1)$ and $F=(0,12,0)$.
 		\end{enumerate}
+		\begin{solution}
+			\begin{enumerate}
+				\item $\vec x=t\mat{2\\0\\1}+s\mat{0\\3\\1}+\mat{2\\0\\0}$ \qquad\qquad\qquad
```

Comment:
What's with the `quad`s? They shouldn't be there. If you need to separate text, use `;` or the word `and`